src/west/commands/project.py

```diff
@@ -398,82 +367,45 @@ def do_run(self, args, user_args):
                                       for p in failed_rebases)))
             raise CommandError(1)
 
-    def _post_checkout_help(self, args, project, branch, sha,
-                            is_ancestor):
-        # Print helpful information to the user about a project that
-        # might have just left a branch behind.
-
-        if branch is None:
-            # If there was no branch checked out, there are no
-            # additional diagnostics that need emitting.
-            return
-
-        relpath = os.path.relpath(project.abspath)
-        if is_ancestor:
-            # If the branch we just left behind is a descendant of
-            # the new HEAD (e.g. if this is a topic branch the
-            # user is working on and the remote hasn't changed),
-            # print a message that makes it easy to get back,
-            # no matter where in the installation os.getcwd() is.
-            _wrn(project,
-                 ('left behind {{name}} branch "{}"; '
-                  'to fast forward back, use: git -C {} checkout {}').
-                 format(branch, relpath, branch))
-            log.dbg('(To do this automatically in the future,',
-                    'use "west update --keep-descendants".)')
-        else:
-            # Tell the user how they could rebase by hand, and
-            # point them at west update --rebase.
-            _wrn(project,
-                 ('left behind {{name}} branch "{}"; '
-                  'to rebase onto the new HEAD: git -C {} rebase {} {}').
-                 format(branch, relpath, sha, branch))
-            log.dbg('(To do this automatically in the future,',
-                    'use "west update --rebase".)')
-
 
 class SelfUpdate(WestCommand):
     def __init__(self):
         super().__init__(
             'selfupdate',
             'selfupdate the west repository',
             _wrap('''
-            Updates the manifest repository and/or the West source code
-            repository. The remote to update from is taken from the
-            manifest.remote and manifest.remote configuration settings, and the
-            revision from manifest.revision and west.revision configuration
-            settings.
+            Updates the West source code repository. The remote to update from
+            is taken from the west.remote configuration setting, and the
+            revision from west.revision configuration setting.
```

Comment:
These are actually in the manifest file, not a configuration file, right? Use of the word "configuration" makes that a bit confusing in my opinion.

Reply:
agree.
Guess I should have cleaned up a bit more of the old text.